ctpolicy/ctpolicy.go

```diff
@@ -85,12 +86,15 @@ func (ctp *CTPolicy) GetSCTs(ctx context.Context, cert core.CertDER) ([]core.SCT
 	results := make(chan result, len(ctp.groups))
 	subCtx, cancel := context.WithCancel(ctx)
 	defer cancel()
-	for _, g := range ctp.groups {
-		go func(g []cmd.LogDescription) {
+	for i, g := range ctp.groups {
+		go func(i int, g []cmd.LogDescription) {
 			sct, err := ctp.race(subCtx, cert, g)
 			// Only one of these will be non-nil
-			results <- result{sct: sct, err: err}
-		}(g)
+			if err != nil {
+				results <- result{err: fmt.Errorf("CT log group %d: %s", i, err)}
```

Comment:
So this is slightly confusing, and will end up requiring a person to go back to the config to figure out which group is actually being talked about. Probably not required to be done in this PR but I think we should actually turn the group description into a struct so that it can have actual name etc.